docs/manual/format.md

```diff
@@ -138,8 +138,14 @@ in the RPM sources.
 
 ## Payload
 
-The Payload is currently a gzipped cpio archive.  The cpio
-archive type used is SVR4 with a CRC checksum.
+The Payload is currently a cpio archive, gzipped by default.  The cpio archive
+type used is SVR4 with a CRC checksum.
```

Comment:
This is missing the format for large files with is a proprietary stripped down version of cpio with most of the metadata missing.

Reply:
Oh that. I wasn't going to fix that in this PR, but let me think about it.